src/ViewHelpers/StagesPageViewHelper.php

```diff
@@ -18,18 +19,54 @@ public static function displayStages($stages)
                 $result .= '<td>';
                 $result .= '<p>'. $stage->getStageTitle().'</p>';
                 $result .= '<form data-id="'. $stage->getStageId().'" class="stagesTableForm">';
-                $result .= '<input type="text" class="stageEditTitle col-xs-10" value="'
-                    . $stage->getStageTitle().'"/>';
+                $result .= '<input type="text" class="stageEditTitle ';
+                $result .= 'col-xs-10" value="'. $stage->getStageTitle().'"/>';
                 $result .= '<input type="submit" class="stageEditSubmit btn-success" value="Submit">';
                 $result .= '</form>';
+                $result .= '<div class="optionsContainer hide" data-stageId="'. $stage->getStageId().'">';
+                if (empty($stage->getOptions())) {
+                    $result .= '<div class="optionContainer multiOptionsContainer">';
+                    $result .= '<form data-id="" class="optionAddForm">';
+                    $result .= '<input type="text" class="optionAddTitle ';
+                    $result .= 'firstOption col-xs-10" placeholder="Type the name of your new option"/>';
+                } else {
+                    foreach ($stage->getOptions() as $option) {
+                        $result .= '<div class="optionContainer">';
+                        $result .= '<p class="optionTitle" data-optionId="';
+                        $result .= $option->getOptionId().'">'. $option->getOptionTitle();
+                        $result .= '<a class="text-danger optionDelete" data-optionId="';
+                        $result .= $option->getOptionId().'">Delete</a>';
+                        $result .= '<a class="optionEdit" data-optionId="'.$option->getOptionId().'">Edit</a>';
+                        $result .= '</p>';
+                        $result .= '<form class="optionTableForm hide" data-optionId="'.$option->getOptionId().'">';
+                        $result .= '<input type="text" class="optionEditTitle col-xs-10" value="';
+                        $result .= $option->getOptionTitle().'"/>';
+                        $result .= '<input type="submit" class="optionEditSubmit btn-success" ';
+                        $result .= 'value="Submit" data-optionid="'.$option->getOptionId().'">';
+                        $result .= '</form>';
+                        $result .= '</div>';
+                    }
+                    $result .= '<div class="optionContainer">';
+                    $result .= '<form data-id="" class="optionAddForm ">';
+                }
+                $result .= '<input type="text" class="optionAddTitle col-xs-10" ';
+                $result .= 'placeholder="Type the name of your new option"/>';
+                $result .= '<input type="submit" class="optionAddSubmit btn-success" data-stageid="';
+                $result .= $stage->getStageId().'" value="Submit">';
+                $result .= '</form>';
```

Comment:
maybe could combine some of these shorter lines? only a thought, just there's a lot of $result .=

Reply:
unfortunately this is to keep below the 120 char limit of circle-ci